### DISCUSSION TITLE
Pls help in configuring multiple environments and how we can promote to different environments.

### DISCUSSION THREAD

**Discussion by kasiviswanathan13** [apache/camel-karavan#1396] [Q&A]
### Ask a question

Pls help in configuring multiple environments and how we can promote to different environments. The documentation just highlights how we can deploy in dev mode, how to productionize this is not mentioned.

### Variant

Web Application

### Container Management (if applicable)

Kubernetes

### Operating System (if applicable)

Linux

### Version

4.7.0

**Comment by mgubaidullin** [comment]
Karavan supports development environment only
